pkgs/by-name/ti/tigerbeetle/package.nix

```diff
@@ -6,20 +6,27 @@
 , tigerbeetle
 , nix-update-script
 }:
+let
+  # Read [these comments](pkgs/development/compilers/zig/hook.nix#L12-L30) on the default Zig flags, and the associated links. tigerbeetle stopped exposing the `-Doptimize` build flag, so we can't use the default Nixpkgs zig hook as-is. tigerbeetle only exposes a boolean `-Drelease` flag which we'll add in the tigerbeetle derivation in this file.
```

Comment:
is this worthy of filing an issue (or patch) upstream to add back support for optimize?

Reply:
reading the PR comment I'd say no.
https://github.com/tigerbeetle/tigerbeetle/pull/1362